plutus-core/cost-model/data/builtinCostModel.json

```diff
@@ -239,13 +253,13 @@
         },
         "cpu": {
             "arguments": {
-                "slope": 1194,
-                "intercept": 251934
+                "slope": 1722,
+                "intercept": 235735
             },
             "type": "max_size"
         }
     },
-    "sha2": {
+    "sha2_256": {
```

Comment:
I was quite worried about this.  Aeson automtically derives toJSON and fromJSON methods via `camelToSnake`, which converts, eg ,`LessThanInteger` to `less_than_integer`. But what happens if your name already has an underscore in it?  The documentation doesn't mention that as far as I can see.  Will `sha2_256` get converted back to `Sha2256`?  Apparently that doesn't happen.  It seems to do the camel to snake conversion using `camelTo2` [here](https://hackage.haskell.org/package/aeson-1.5.6.0/docs/src/Data.Aeson.Types.Internal.html#camelTo2), and that just ignores underscores.  I don't understand all of the Aeon code, but I don't think there's a conversion in the other direction.  I think when it's doing `fromJSON` it may convert the name of the thing it's looking for into snake case and then look for that in the encoded object, so there won't be a problem.  In any case, it appears to work.